packages/pwa/app/components/image/index.jsx

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Image as ChakraImage, Img as ChakraImg} from '@chakra-ui/react'
+import theme from '@chakra-ui/theme'
+
+const propTypes = {
+    src: PropTypes.string,
+    vwSizes: PropTypes.arrayOf(PropTypes.number),
+    sizes: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.string),
+        PropTypes.object,
+        PropTypes.string
+    ]),
+    srcSet: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.number), PropTypes.string])
+}
+
+export const Image = ({src, vwSizes, sizes, srcSet, ...otherProps}) => {
+    return <ChakraImage {...getImageProps({src, vwSizes, sizes, srcSet, ...otherProps})} />
+}
+Image.propTypes = propTypes
```

Comment:
A few comments on this block: 

1. It looks like we are passing `otherProps` to the `getImageProps` utility function where it's unused, is there a specific reason we are passing it though? 
2. We are also passing a prop called `srcSet` which is modified and returned with the same name. Is there a better name for this prop so it's easily differentiated from the value (aptly named srcSet because it is a html srcSet) that comes back?
3. I renamed `getImageProps` to `getResponsiveImageAttributes` to be a little more descriptive. But thats just a suggestion.

```suggestion
export const Image = ({src, vwSizes, sizes, srcSet, ...rest}) => {
    return <ChakraImage {...rest} {...getResponsiveImageAttributes({src, vwSizes, sizes, srcSet})} />
}
Image.propTypes = propTypes
```

Reply:
Ok, I've committed your suggestions for 1 and 3.

Reply:
For 2: I was thinking too of a different prop name, but decided that `srcSet` is a good name already and its value can be represented in different formats, similar to [Chakra's responsive styles](https://chakra-ui.com/docs/features/responsive-styles#the-array-syntax). 

I see our Image component as an extension of Chakra's Image component. And in Chakra, for example, style props like `width` can be represented in any of these values:
- literal string → it will pass through without modification
- array
- object

Reply:
On second thought, I think you got a point there (for number 2). How about renaming the props to these?

- `sizesForBreakpoints` → mapped to `sizes` - I think now it's clearer that the sizes are actually relative to the breakpoints
- `srcSetWidths` → mapped to `srcSet`

Reply:
And about the existing `vwSizes`: I now think it's not necessary and we can remove it to make things clearer.

Instead of `vwSizes={[100, 100, 50]}`:
```jsx
sizesForBreakpoints={['100vw', '100vw', '50vw']}
// If `srcSet` or `srcSetWidths` are not given, we would automatically calculate it for them
```